plasma-sdk/packages/privy-auth/src/hooks.ts

```
@@ -40,6 +40,67 @@ import type {
+  logout: async () => {},
+  linkEmail: () => {},
+  linkPhone: () => {},
+} as unknown as ReturnType<typeof usePrivy>;
+
+const EMPTY_WALLETS = { wallets: [] } as unknown as ReturnType<
+  typeof useWallets
+>;
+
+function useSafePrivy(): ReturnType<typeof usePrivy> {
+  try {
+    return usePrivy();
```

Biome (2.3.14)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)